tests/run_all_tests.py

```
@@ -0,0 +1,299 @@
+    resultclass = ColoredTextTestResult
+
+
+def load_test_module(module_name):
+    """Load a test module by name"""
+    try:
+        spec = importlib.util.spec_from_file_location(
+            module_name, 
+            os.path.join(os.path.dirname(__file__), f"{module_name}.py")
+        )
+        if spec is None or spec.loader is None:
+            raise ImportError(f"Cannot load spec for {module_name}")
```

Ruff (0.14.4)

Abstract `raise` to an inner function

(TRY301)